### PR TITLE
Interjections should not force preanimation if 'Pre' is unchecked

### PR DESCRIPTION
**Pull request by in1tiate**

### CODE REVIEW

File: src/courtroom.cpp [hunk 1]
```diff
@@ -1347,6 +1347,13 @@ void Courtroom::on_chat_return_pressed()
       else
         f_emote_mod = 2;
     }
+    else
+    {
+      if (f_emote_mod == 1)
+        f_emote_mod = 0;
+      else if (f_emote_mod == 4)
+        f_emote_mod = 5;
+    }
```
Comment (oldmud0) [inline src/courtroom.cpp:1356]: Seems contrived. Probably this if block should be condensed to `if (objection_state >0 && ui_pre->isChecked())`. Then the control flow can go to the `else` block at the very bottom.

Comment (in1tiate) [reply src/courtroom.cpp]: Done (002c940), good catch.

**Comment by in1tiate** [comment]
Already implemented by the 2.7 merge.

**Comment by in1tiate** [comment]
Not implemented in 2.8.5.

**Pull request closed without merging**